lib/src/client.ts

```diff
@@ -283,14 +283,19 @@ export class AsgardeoAuthClient<T> {
         authorizationCode: string,
         sessionState: string,
         state: string,
-        userID?: string
+        userID?: string,
+        tokenRequestConfig?: {
```

Comment:
Lets update the `README` as well with the new method signature 💪 

- https://github.com/asgardeo/asgardeo-auth-js-core/blob/9769b04bd5fb274d21a8c59ebf8318933e896a1b/README.md?plain=1#L415